### PR TITLE
Legally Distinct Orange Cat + capybara fix

### PR DESCRIPTION
**Pull request by leonardo-dabepis**
## About the PR

Adds a Legally Distinct orange cat, makes capybaras available to order from cargo, adds sprites for breath masks and 02 tanks for both creatures. Mappers can also place spawn points for capybaras and orange cats, and the cat crate probably has a small chance of spawning an orange cat unless I coded it wrong but I am so tired

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added orange cat, capybara crate, 02/tank sprites for orange cat and capybara, orange cat and capybara spawners.
- remove: Removed a brain cell

### CODE REVIEW

**Comment by Plyushsune** [comment]
Strange I been seeing that rsi error on my prs too

File: Resources/Textures/DeltaV/Objects/Tanks/emergency.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      "name": "equipped-SUITSTORAGE-capybara",
+      "directions": 4,
+      "delays": [[1],[1],[1],[1]]
+    },
+    {
+      "name": "equipped-SUITSTORAGE-catorange",
+      "directions": 4,
+      "delays": [[1],[1],[1],[1]]
+    },
+    }
```
Comment (FluffiestFloof) [inline Resources/Textures/DeltaV/Objects/Tanks/emergency.rsi/meta.json]: Remove the additional bracket at line 20 and the comma after the bracket of line 19.

File: Resources/Textures/DeltaV/Clothing/Mask/breath.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "states": [
+    {
+      "name": "equipped-MASK-capybara",
+      "directions": 4
+    },
+    {
+      "name": "up-equipped-MASK-catorange",
+      "directions": 4
+    },
+    }
```
Comment (FluffiestFloof) [inline Resources/Textures/DeltaV/Clothing/Mask/breath.rsi/meta.json]: Remove the additional bracket at line 18 and the comma after the bracket of line 17.

**Review by FluffiestFloof — COMMENTED** [review]
The RSI validator is mainly not happy about things you haven't touched and those two issues.
Pretty much replacing those two line with just a simple closing bracket.

It's also not happy about things that are unrelated, like cluwne???

tldr, you added a closing bracket that doesn't close any other open bracket and you added a comma that tells the file "hey I have something else for you" without having something else

**Comment by DebugOk** [comment]
I put out a quick fix for the rsi validator, if you update your branch, it should now only show errors relating to your changes

**Comment by leonardo-dabepis** [comment]
> The RSI validator is mainly not happy about things you haven't touched and those two issues. Pretty much replacing those two line with just a simple closing bracket.
> 
> It's also not happy about things that are unrelated, like cluwne???
> 
> tldr, you added a closing bracket that doesn't close any other open bracket and you added a comma that tells the file "hey I have something else for you" without having something else

oh, those two files in particular I think I forgot to delete because I ended up just modifying the preexisting breath mask and gas tank from wizden to fit on my creatures. I figured that's probably the way to go since I'm just making it so my creatures can use an already existing item, rather than making a new item for them entirely. I fixed the files but I will still keep the ones in the DeltaV folder until told otherwise. But those ones in the DeltaV folder are probably redundant at this point. This whole thing has given me a headache :)
